packages/devtools_app/lib/src/screens/memory/panes/diff/controller/diff_pane_controller.dart

```diff
@@ -184,13 +175,32 @@ class DerivedData extends DisposableController with AutoDisposeControllerMixin {
   DerivedData(this._core) {
     _selectedItem = ValueNotifier<SnapshotItem>(_core.selectedItem);
 
+    final classFilterData = ClassFilterData(
+      filter: _core.classFilter,
+      onChanged: applyFilter,
+      rootPackage: serviceManager.rootInfoNow().package,
```

Comment:
will this value ever change throughout the lifetime of an app connection?

Reply:
nope. removed from constructor